src/Listener/JsonApiListener.php

```diff
@@ -197,6 +205,19 @@ protected function _validateConfigOptions()
     }
 
     /**
+     * Make sure the neomerx/json-api composer package is installed
+     *
+     * @throws \Crud\Error\Exception\CrudException
+     * @return void
+     */
+    protected function _checkPackageDependencies()
+    {
+        if (!class_exists('\Neomerx\JsonApi\Encoder\Encoder')) {
+            throw new CrudException('JsonApiListener requires composer installing' . $this->neomerxPackage);
```

Comment:
it was more related to me face palming over stupid mistakes caused by trying to squeeze in commits in between things 😱